app/Concerns/RedirectToInitialEvaluation.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Concerns;
+
+trait RedirectToInitialEvaluation
+{
+    protected function getRedirectUrl(): string
+    {
+        return self::$resource::getUrl('view_initial_evaluation', ['record' => $this->record->id]) .
+            '?tab=-' . $this->getTabSlug() . '-tab';
+    }
```

Comment:
nu ai nevoie sa faci concat mai pune un atribut tab, si daca e nu declar in ruta il pune el automat cu query parm. 
Si foloseste sprintf() nu fac concaternari